services/models.py

```diff
@@ -1,13 +1,42 @@
+from adminsortable.models import SortableMixin
 from django.db import models
+from django.db.models import Max
 from enumfields import EnumField
+from parler.models import TranslatableModel, TranslatedFields
 
 from profiles.models import Profile
 
 from .enums import ServiceType
 
 
-class Service(models.Model):
+def get_next_data_field_order():
+    try:
+        return AllowedDataField.objects.all().aggregate(Max("order"))["order__max"] + 1
+    except TypeError:
+        return 1
+
+
+class AllowedDataField(TranslatableModel, SortableMixin):
+    field_name = models.CharField(max_length=30)
+    translations = TranslatedFields(label=models.CharField(max_length=64))
+    order = models.PositiveIntegerField(
+        default=get_next_data_field_order, editable=False, db_index=True
```

Comment:
this is not meant to change, right? once you create an object, you cannot change its ~priority?

Reply:
It's meant to change but we don't want to show it on a form. There's a drag & drop UI for changing it:

![Screenshot 2020-01-28 at 13 10 26](https://user-images.githubusercontent.com/4736847/73259604-cf96f000-41d0-11ea-8c49-dd7221efce06.png)

Reply:
ah, ok, that package will handle it. gotcha 👍